File: Functions/Rotation.cpp
#include<iostream>
#include<cmath>
#include</Users/sotiriskakaletsis/Documents/GitHub/isofin/Eigen/Eigen/Dense>
//#include <C:\Users\smm5969\Box Sync\Fibrin CPP\Functions\cross_vM.cpp>
using namespace Eigen;
using namespace std;
Matrix3d Rotation( Vector3d &t, double psi )
{
	Matrix3d matrix,I; 
	I = Matrix3d::Identity();
	matrix = I*cos(psi)+sin(psi)*cross_vM(t,I);
	return matrix;
}

File: Matlab/Formulation/Functions/Rotation.m
function [ matrix ] = Rotation( t,psi )
%UNTITLED Summary of this function goes here
%   Detailed explanation goes here
%% Code
% I=eye(3);
% matrix=I*cos(psi)+sin(psi)*cross_vM(t,I);% Rotation matrix.
I=[1,0,0;0,1,0;0,0,1];matrix=I*cos(psi)+sin(psi)*cross_vM(t,I);% Rotation matrix.
end

